core/src/main/resources/error/error-classes.json

```diff
@@ -138,6 +153,9 @@
   "UNABLE_TO_ACQUIRE_MEMORY" : {
     "message" : [ "Unable to acquire %s bytes of memory, got %s" ]
   },
+  "UNSUPPORTED_FROM_TO_INTERVAL" : {
```

Comment:
Could you revisit the error classes you added, and try to reuse existing classes. For example, this one `UNSUPPORTED_FROM_TO_INTERVAL` is not needed. You could use `UNSUPPORTED_FEATURE`

Reply:
Sure

Reply:
@MaxGekk Test queries are throwing  messages similar to "Intervals FROM month TO second are not supported." and hence using UNSUPPORTED_FEATURE is leading to test failures in many places. So I think using UNSUPPORTED_FROM_TO_INTERVAL will be a good option and also it matches the exactly error messages thrown by the queries.

Reply:
@MaxGekk   Error message "Intervals FROM * TO * are not supported" is thrown from many places(AstBuilder, Interval_sql etc.) so if we reuse UNSUPPORTED_FEATURE  as an alternate to UNSUPPORTED_FROM_TO_INTERVAL , then tests are failed in other test suits like "ExpressionParserSuite" which is not related to this jira. So using  UNSUPPORTED_FROM_TO_INTERVAL error class will be suitable and also we can avoid other test case failures.

Reply:
> then tests are failed in other test suits like "ExpressionParserSuite"

I think we should move such tests to QueryParsingErrorsSuite, QueryCompilationErrorsSuite or QueryExecutionErrorsSuite when ever it is possible.

> So using UNSUPPORTED_FROM_TO_INTERVAL error class will be suitable and also we can avoid other test case failures.

After offline discussion with @cloud-fan, we agreed to make our best efforts and use existing **general** error classes. See this PR, for instance https://github.com/apache/spark/pull/35302

Reply:
@MaxGekk  As per the above input, I have moved test query from ExpressionParserSuite.scala and Interval.sql  to QueryParsingErrorSuite.

Reply:
@MaxGekk I have Modified out /ansi/interval.sql.out and results/interval.sql.out files in order to avoid test failure of query "select interval '1' year to second".